### PR TITLE
[IMP] Market Connectors: eBay marketplace edits.

### PR DESCRIPTION
**Pull request by road-odoo**
Added how to configure the policies of the product in order to prevent the error ```Input data for tag <Item.SellerProfiles.SellerPaymentProfile.PaymentProfileID> is invalid or missing.```from appearing when the user is listing the product. Also changed the section of listing images to indicate that they are necessary to prevent error ```A description is required.(106)Add at least 1 photo. More photos are better! Show off your item from every angle and zoom in on details.(21919136)``` from appearing. The first error is really common when a user is listing products and there have been multiple help tickets opened for the same reason

### CODE REVIEW

**Review by pabr-odoo — APPROVED** [review]
Changes look good! Passing it off to @ksc-odoo for final review.

File: content/applications/sales/sales/ebay_connector/manage.rst
Comment (ksc-odoo) [inline content/applications/sales/sales/ebay_connector/manage.rst:51]:
```suggestion
The policies need to be set, in order to be able to list on eBay. A value must be selected for the
```

File: content/applications/sales/sales/ebay_connector/manage.rst
Comment (ksc-odoo) [inline content/applications/sales/sales/ebay_connector/manage.rst:52]:
```suggestion
fields: :guilabel:`Payment Policy`, :guilabel:`Return Policy`, and :guilabel:`Shipping Policy`. The options
```

File: content/applications/sales/sales/ebay_connector/manage.rst
Comment (ksc-odoo) [inline content/applications/sales/sales/ebay_connector/manage.rst:53]:
```suggestion
are imported from the eBay database. If no option appears, they can be imported
```

File: content/applications/sales/sales/ebay_connector/manage.rst
Comment (ksc-odoo) [inline content/applications/sales/sales/ebay_connector/manage.rst:54]:
```suggestion
by navigating to :menuselection:`Sales app --> Configuration --> Settings --> eBay section`, and click on the
```

File: content/applications/sales/sales/ebay_connector/manage.rst
Comment (ksc-odoo) [inline content/applications/sales/sales/ebay_connector/manage.rst:56]:
```suggestion
:guilabel:`Policies` button in the :guilabel:`Synchronization` field. This button **only** appears
once the production and sandbox credentials have been set.
```

File: content/applications/sales/sales/ebay_connector/manage.rst
Comment (ksc-odoo) [inline content/applications/sales/sales/ebay_connector/manage.rst:59]:
```suggestion
default template only uses the :guilabel:`eBay Description` field of the product. HTML can be used
```

File: content/applications/sales/sales/ebay_connector/manage.rst
Comment (ksc-odoo) [inline content/applications/sales/sales/ebay_connector/manage.rst:60]:
```suggestion
inside the :guilabel:`Description Template`, and in the :guilabel:`eBay Description` in Odoo 14.
```

File: content/applications/sales/sales/ebay_connector/manage.rst
Comment (ksc-odoo) [inline content/applications/sales/sales/ebay_connector/manage.rst:65]:
```suggestion
To use images in the listing, an image needs to be added as one of the *Attachments* on the product
```

**Review by ksc-odoo — APPROVED** [review]
@road-odoo -- alrighty - just finished my Final Review. Approving now. Once you implement the necessary adjustments, feel free to move it to the next stage. Thanks.

**Comment by StraubCreative** [comment]
@samueljlieber tagging you on here so it's on your radar, however FR round CR's still need to be addressed and then it should be ready for you.

cc: @road-odoo

File: content/applications/sales/sales/ebay_connector/manage.rst
Comment (ksc-odoo) [inline content/applications/sales/sales/ebay_connector/manage.rst:48]:
```suggestion
When the :guilabel:`Use Stock Quantity` field is checked, the quantity set on eBay is the Odoo
```

File: content/applications/sales/sales/ebay_connector/manage.rst
Comment (ksc-odoo) [inline content/applications/sales/sales/ebay_connector/manage.rst:51]:
```suggestion
The policies need to be set in order to be able to list on eBay. A value must be selected for the
```

File: content/applications/sales/sales/ebay_connector/manage.rst
Comment (ksc-odoo) [inline content/applications/sales/sales/ebay_connector/manage.rst:52]:
```suggestion
following fields: :guilabel:`Payment Policy`, :guilabel:`Return Policy`, and :guilabel:`Shipping Policy`. The options
```

File: content/applications/sales/sales/ebay_connector/manage.rst
Comment (ksc-odoo) [inline content/applications/sales/sales/ebay_connector/manage.rst:54]:
```suggestion
:menuselection:`Sales app --> Configuration --> Settings --> eBay`, and clicking on the :guilabel:`Policies`
```

File: content/applications/sales/sales/ebay_connector/manage.rst
Comment (ksc-odoo) [inline content/applications/sales/sales/ebay_connector/manage.rst:55]:
```suggestion
button in the :guilabel:`Synchronization` field. This button **only** appears once the production
```

**Review by ksc-odoo — APPROVED** [review]
hey there @road-odoo -- just a handful of super quick adjustments need to be made. but, once you take care of those necessary adjustments, you can *then* move this to the next round. thanks! 👍

**Comment by StraubCreative** [comment]
@road-odoo peer and final review are complete (we don't need multiple rounds of same reviewer for such small adjustments), so can move to tech review now with @samueljlieber. Thank you for your work and attention here!

**Review by samueljlieber — APPROVED** [review]
Hi @road-odoo! Thank you for your work on this PR, your improvements look good to me! I am approving with a small fix that I will push up just after my review, and merge. 

Also update the commit message to include the app/module name :)
```
[IMP] sales: market connectors eBay marketplace
```

Thank you!

**Comment by samueljlieber** [comment]
@robodoo r+

**Pull request closed without merging**
